### DISCUSSION TITLE
Essentials compiling for arch 7.5 even when setting -DCMAKE_CUDA_ARCHITECTURES=70

### DISCUSSION THREAD

**Discussion by alexfallin** [gunrock/gunrock#1040] [Q&A] [answered]
Hello,

I'm not sure that this is the correct place to put this. I am attempting to run Gunrocks MST to compare to an implementation I am working on. The oldest GPU we're running the experiments on is a TITAN V which is arch 7.0. I attempted to set -DCMAKE_CUDA_ARCHITECTURES=70 but it is still compiling the essentials using arch 7.5:
![image](https://user-images.githubusercontent.com/32699361/228389473-80ed0666-7f71-4c57-8fe8-dd2d92e11e9e.png)
This is causing an error where thrust does not have a kernel image for the older device. Is there a way I can force everything to compile for 7.0, or is this unsupported?

Thanks,
Alex

**Comment by neoblizz** [comment, marked as answer]
Hi Alex,

This is an issue, I'll move it there and an easy fix! I will add the fix to the `main` branch. See: https://github.com/gunrock/gunrock/issues/1041
